adafruit_circuitplayground/circuit_playground_base.py

```diff
@@ -686,19 +692,19 @@ def red_led(self):
         return self._led.value
 
     @red_led.setter
-    def red_led(self, value):
+    def red_led(self, value) -> None:
         self._led.value = value
 
     @staticmethod
-    def _sine_sample(length):
+    def _sine_sample(length) -> None:
```

Comment:
You can type annotate `length` as `int`.  Additionally, this is a generator that only yields values, so we can use `typing.Iterator` to type annotate the return.  So importing `Iterator` from `typing`, we can do:

```suggestion
    def _sine_sample(length: int) -> Iterator[int]:
```